elastalert/elastalert.py

```diff
@@ -1121,6 +1128,7 @@ def handle_error(self, message, data=None):
 
     def handle_uncaught_exception(self, exception, rule):
         """ Disables a rule and sends a notifcation. """
+        logging.debug(traceback.format_exc())
```

Comment:
I continue to believe it's surprising that these errors show up in logs other than error (violating the principle of least astonishment).

As long as it's documented within elastalert and internally consistent, however, this is tolerable.